### PR TITLE
framework/st_things : code clean, remove accesspointlist resource

### PR DESCRIPTION
**Pull request by JeongJunSik**
- unuse code cleanup
- remove accesspointlist resource
- rename "OC_" to "THINGS_"
- change the system time setting position (resource handler -> app callback)
- apply coding rule

### CODE REVIEW

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c [hunk 1]
```diff
@@ -39,6 +39,8 @@
 #include "utils/things_thread.h"
 #include "utils/things_rtos_util.h"
 
+#include <time.h>
```
Comment (sunghan-chang) [inline framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: Please move this at location of standard header including. (line 29)

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c [hunk 2]
```diff
@@ -727,6 +730,58 @@ void dev_conf_prov_cb_in_app(es_dev_conf_prov_data_s *event_data)
 		// check payload
 		THINGS_LOG_D(THINGS_DEBUG, TAG, "Datetime : %s", event_data->datetime);
 	}
+
+	//Added by THINGS to set time in appliance yyyy-mm-ddThh-mm-ss
+	char chYear[5], chMonth[3], chDay[3], chHour[3], chMin[3], chSec[3];
```
Comment (sunghan-chang) [inline framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: every variable should be defined at new line. That is our coding rule.

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c (hunk 1 above)
Comment (JeongJunSik) [reply framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: OK Thanks.
I will fix it

File: framework/src/st_things/things_stack/easy-setup/easysetup_manager.c (hunk 2 above)
Comment (JeongJunSik) [reply framework/src/st_things/things_stack/easy-setup/easysetup_manager.c]: OK Thanks.
I will fix it

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**